linux_files/setup

```diff
@@ -648,6 +654,24 @@ else
 fi
 }
 
+function sysinternalsinstall {
+
+if (whiptail --title "SYSINTERNALS" --yesno "Would you like to download and install Linux ports of Microsoft Sysinternals tools?" 8 90) then
+    echo "Installing SYSINTERNALS"
+    createtmp
+    curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+    sudo cp microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
+    rm microsoft.gpg
+    sudo sh -c 'echo "deb https://packages.microsoft.com/repos/microsoft-ubuntu-xenial-prod stretch main" > /etc/apt/sources.list.d/microsoft.list' 
```

Comment:
Will this be collide with powershell installation? it uses `microsoft-debian-stretch-prod`

Reply:
If you install Sysinternals and then Powershell, then yes, Sysinternals wont get updated. I have opened an issue with Microrsoft to get it added to their Debian repo for Stretch.